### ISSUE TITLE
iOS issue

### ISSUE THREAD

**Issue by Talha-Jawed** [lesliesam/react-native-wheel-picker#78]
Simply not working on iOS, maybe you need to upgrade the iOS system.

<img width="355" alt="Screenshot 2021-04-06 at 4 04 06 PM" src="https://user-images.githubusercontent.com/39768820/113701542-bb625300-96f1-11eb-842a-7935880b3f73.png">

**- Platform that you're experiencing the issue on:**

- iOS

**Comment by tinydiv333** [comment]
Have you find any solution?

**Comment by Talha-Jawed** [comment]
@tinydiv333  yes first you install `@react-native-picker/picker` and then install `@gregfrench/react-native-wheel-picker`

**STEPS:**
1- `yarn remove @gregfrench/react-native-wheel-picker`
2- delete node_modules folder
3- `yarn add @react-native-picker/picker`
4- `yarn add @gregfrench/react-native-wheel-picker`

It`s work for me
